.changeset/selfish-balloons-work.md

```diff
@@ -0,0 +1,8 @@
+---
+"@khanacademy/math-input": major
```

Comment:
Is `math-input` really having a major version change due to these changes?

Reply:
I think I did that because I renamed/retyped some thing in the KeypadContext which is outward facing - so I thought that might constitute a breaking change. When I think major/minor, I usually think: "could someone bump the version without changing their code?" This is in the grey area: _maybe_ they wouldn't have to? But I'm not 100% sure.